### DISCUSSION TITLE
/storage shows empty with user, but full with user/group 0

### DISCUSSION THREAD

**Discussion by tpmehta** [jlesage/docker-crashplan-pro#466] [Q&A]
Hi all, I'm running this container on a Synology NAS, and am running into the issue with my mapped /storage location being empty with my user running the container.  When I change to running the container as root, I see the files.  I followed the troubleshooting guide and chose the UID and GID for the user running the container, and restarted multiple times to no avail.  

Any other tips I could try?

If helpful, here are the perms for the mapped folder on my Synology:
drwxrwxrwx+  1 root       root

**Comment by jlesage** [comment]
And what about permissions of files under this folder ?
